src/components/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,60 @@
+import { type VariantProps, cva } from 'class-variance-authority';
+import { cn } from '@/utils/styles';
+
+type CheckBoxProps = VariantProps<typeof CheckBoxVariants> & {
+  className?: string;
+  onCheckedChange: (checked: boolean) => void;
+} & React.ComponentPropsWithoutRef<'input'>;
+
+function CheckBox({
+  className,
+  size1,
+  disabled,
+  checked,
+  onCheckedChange,
+  ...props
+}: CheckBoxProps) {
+  return (
+    <input
+      type="checkbox"
+      checked={checked}
+      disabled={disabled}
+      className={cn(CheckBoxVariants({ size1, disabled, checked }), className)}
+      onChange={e => {
+        onCheckedChange(e.target.checked);
+      }}
+      {...props}
+    >
+      CheckBox
+    </input>
```

Comment:
```suggestion
      }}
      {...props}
    />
```

위와 같이 작성해주시면 Children 문제가 발생하지 않습니다